api/v1/server/handlers/metadata/get.go

```diff
@@ -21,6 +21,10 @@ func (u *MetadataService) MetadataGet(ctx echo.Context, request gen.MetadataGetR
 		authTypes = append(authTypes, "github")
 	}
 
+	if u.config.Auth.ConfigFile.Custom.Enabled {
+		authTypes = append(authTypes, "custom")
```

Comment:
For a small UI improvement, it would be great to be able to provide a name and an icon URL which can be sent to the UI in `login/index.tsx`